### PR TITLE
Updates for Node.js v7.10.0, v6.10.3 and v4.8.3

### PR DESCRIPTION
**Pull request by pesho**
Related: https://github.com/nodejs/docker-node/pull/389
Related: https://github.com/nodejs/docker-node/pull/390
Related: https://github.com/nodejs/docker-node/pull/391
Related: https://github.com/nodejs/docker-node/pull/392

### CODE REVIEW

File: library/node [hunk 1]
```diff
@@ -3,63 +3,63 @@
 Maintainers: The Node.js Docker Team <https://github.com/nodejs/docker-node> (@nodejs)
 GitRepo: https://github.com/nodejs/docker-node.git
 
-Tags: 7.9.0, 7.9, 7, latest
-GitCommit: e1103db1e7330f620ec4b5961b93936da11becdf
-Directory: 7.9
+Tags: 7.10.0, 7.10
```
Comment (yosifkit) [inline library/node]: Looks like `latest` tag got lost; which also means `7-*`, `onbuild`, `slim`, and `wheezy` are missing too.

**Comment by pesho** [comment]
@yosifkit apologies, should be fixed now.

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-list b/_bashbrew-list
index 0290cf3..d46c22d 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -8,11 +8,11 @@ node:4.8-alpine
 node:4.8-onbuild
 node:4.8-slim
 node:4.8-wheezy
-node:4.8.2
-node:4.8.2-alpine
-node:4.8.2-onbuild
-node:4.8.2-slim
-node:4.8.2-wheezy
+node:4.8.3
+node:4.8.3-alpine
+node:4.8.3-onbuild
+node:4.8.3-slim
+node:4.8.3-wheezy
 node:6
 node:6-alpine
 node:6-onbuild
@@ -23,26 +23,26 @@ node:6.10-alpine
 node:6.10-onbuild
 node:6.10-slim
 node:6.10-wheezy
-node:6.10.2
-node:6.10.2-alpine
-node:6.10.2-onbuild
-node:6.10.2-slim
-node:6.10.2-wheezy
+node:6.10.3
+node:6.10.3-alpine
+node:6.10.3-onbuild
+node:6.10.3-slim
+node:6.10.3-wheezy
 node:7
 node:7-alpine
 node:7-onbuild
 node:7-slim
 node:7-wheezy
-node:7.9
-node:7.9-alpine
-node:7.9-onbuild
-node:7.9-slim
-node:7.9-wheezy
-node:7.9.0
-node:7.9.0-alpine
-node:7.9.0-onbuild
-node:7.9.0-slim
-node:7.9.0-wheezy
+node:7.10
+node:7.10-alpine
+node:7.10-onbuild
+node:7.10-slim
+node:7.10-wheezy
+node:7.10.0
+node:7.10.0-alpine
+node:7.10.0-onbuild
+node:7.10.0-slim
+node:7.10.0-wheezy
 node:alpine
 node:argon
 node:argon-alpine
diff --git a/node_alpine/Dockerfile b/node_alpine/Dockerfile
index 4eeb943..5269915 100644
--- a/node_alpine/Dockerfile
+++ b/node_alpine/Dockerfile
@@ -1,7 +1,7 @@
 FROM alpine:3.4
 
 ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_VERSION 7.9.0
+ENV NODE_VERSION 7.10.0
 
 RUN addgroup -g 1000 node \
     && adduser -u 1000 -G node -s /bin/sh -D node \
@@ -46,7 +46,7 @@ RUN addgroup -g 1000 node \
     && rm -Rf "node-v$NODE_VERSION" \
     && rm "node-v$NODE_VERSION.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt
 
-ENV YARN_VERSION 0.23.2
+ENV YARN_VERSION 0.23.4
 
 RUN apk add --no-cache --virtual .build-deps-yarn curl gnupg \
   && for key in \
diff --git a/node_argon-alpine/Dockerfile b/node_argon-alpine/Dockerfile
index a530c2b..07bb275 100644
--- a/node_argon-alpine/Dockerfile
+++ b/node_argon-alpine/Dockerfile
@@ -1,7 +1,7 @@
 FROM alpine:3.4
 
 ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_VERSION 4.8.2
+ENV NODE_VERSION 4.8.3
 
 RUN addgroup -g 1000 node \
     && adduser -u 1000 -G node -s /bin/sh -D node \
@@ -46,7 +46,7 @@ RUN addgroup -g 1000 node \
     && rm -Rf "node-v$NODE_VERSION" \
     && rm "node-v$NODE_VERSION.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt
 
-ENV YARN_VERSION 0.23.2
+ENV YARN_VERSION 0.23.4
 
 RUN apk add --no-cache --virtual .build-deps-yarn curl gnupg \
   && for key in \
diff --git a/node_argon-onbuild/Dockerfile b/node_argon-onbuild/Dockerfile
index 1afb96c..d2389e4 100644
--- a/node_argon-onbuild/Dockerfile
+++ b/node_argon-onbuild/Dockerfile
@@ -1,4 +1,4 @@
-FROM node:4.8.2
+FROM node:4.8.3
 
 RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
diff --git a/node_argon-slim/Dockerfile b/node_argon-slim/Dockerfile
index 83f8b15..71a8d8e 100644
--- a/node_argon-slim/Dockerfile
+++ b/node_argon-slim/Dockerfile
@@ -21,7 +21,7 @@ RUN set -ex \
   done
 
 ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_VERSION 4.8.2
+ENV NODE_VERSION 4.8.3
 
 RUN buildDeps='xz-utils' \
     && set -x \
@@ -36,7 +36,7 @@ RUN buildDeps='xz-utils' \
     && apt-get purge -y --auto-remove $buildDeps \
     && ln -s /usr/local/bin/node /usr/local/bin/nodejs
 
-ENV YARN_VERSION 0.23.2
+ENV YARN_VERSION 0.23.4
 
 RUN set -ex \
   && for key in \
diff --git a/node_argon-wheezy/Dockerfile b/node_argon-wheezy/Dockerfile
index 8ee71fb..59bf193 100644
--- a/node_argon-wheezy/Dockerfile
+++ b/node_argon-wheezy/Dockerfile
@@ -21,7 +21,7 @@ RUN set -ex \
   done
 
 ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_VERSION 4.8.2
+ENV NODE_VERSION 4.8.3
 
 RUN curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-x64.tar.xz" \
   && curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/SHASUMS256.txt.asc" \
@@ -31,7 +31,7 @@ RUN curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-
   && rm "node-v$NODE_VERSION-linux-x64.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt \
   && ln -s /usr/local/bin/node /usr/local/bin/nodejs
 
-ENV YARN_VERSION 0.23.2
+ENV YARN_VERSION 0.23.4
 
 RUN set -ex \
   && for key in \
diff --git a/node_argon/Dockerfile b/node_argon/Dockerfile
index f2dd99e..1b44a76 100644
--- a/node_argon/Dockerfile
+++ b/node_argon/Dockerfile
@@ -21,7 +21,7 @@ RUN set -ex \
   done
 
 ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_VERSION 4.8.2
+ENV NODE_VERSION 4.8.3
 
 RUN curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-x64.tar.xz" \
   && curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/SHASUMS256.txt.asc" \
@@ -31,7 +31,7 @@ RUN curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-
   && rm "node-v$NODE_VERSION-linux-x64.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt \
   && ln -s /usr/local/bin/node /usr/local/bin/nodejs
 
-ENV YARN_VERSION 0.23.2
+ENV YARN_VERSION 0.23.4
 
 RUN set -ex \
   && for key in \
diff --git a/node_boron-alpine/Dockerfile b/node_boron-alpine/Dockerfile
index 6815b2a..d0e3388 100644
--- a/node_boron-alpine/Dockerfile
+++ b/node_boron-alpine/Dockerfile
@@ -1,7 +1,7 @@
 FROM alpine:3.4
 
 ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_VERSION 6.10.2
+ENV NODE_VERSION 6.10.3
 
 RUN addgroup -g 1000 node \
     && adduser -u 1000 -G node -s /bin/sh -D node \
@@ -46,7 +46,7 @@ RUN addgroup -g 1000 node \
     && rm -Rf "node-v$NODE_VERSION" \
     && rm "node-v$NODE_VERSION.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt
 
-ENV YARN_VERSION 0.23.2
+ENV YARN_VERSION 0.23.4
 
 RUN apk add --no-cache --virtual .build-deps-yarn curl gnupg \
   && for key in \
diff --git a/node_boron-onbuild/Dockerfile b/node_boron-onbuild/Dockerfile
index 3395e67..c7f91ca 100644
--- a/node_boron-onbuild/Dockerfile
+++ b/node_boron-onbuild/Dockerfile
@@ -1,4 +1,4 @@
-FROM node:6.10.2
+FROM node:6.10.3
 
 RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
diff --git a/node_boron-slim/Dockerfile b/node_boron-slim/Dockerfile
index 73d0f02..ea74fb6 100644
--- a/node_boron-slim/Dockerfile
+++ b/node_boron-slim/Dockerfile
@@ -21,7 +21,7 @@ RUN set -ex \
   done
 
 ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_VERSION 6.10.2
+ENV NODE_VERSION 6.10.3
 
 RUN buildDeps='xz-utils' \
     && set -x \
@@ -36,7 +36,7 @@ RUN buildDeps='xz-utils' \
     && apt-get purge -y --auto-remove $buildDeps \
     && ln -s /usr/local/bin/node /usr/local/bin/nodejs
 
-ENV YARN_VERSION 0.23.2
+ENV YARN_VERSION 0.23.4
 
 RUN set -ex \
   && for key in \
diff --git a/node_boron-wheezy/Dockerfile b/node_boron-wheezy/Dockerfile
index 0b36853..f5f65f1 100644
--- a/node_boron-wheezy/Dockerfile
+++ b/node_boron-wheezy/Dockerfile
@@ -21,7 +21,7 @@ RUN set -ex \
   done
 
 ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_VERSION 6.10.2
+ENV NODE_VERSION 6.10.3
 
 RUN curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-x64.tar.xz" \
   && curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/SHASUMS256.txt.asc" \
@@ -31,7 +31,7 @@ RUN curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-
   && rm "node-v$NODE_VERSION-linux-x64.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt \
   && ln -s /usr/local/bin/node /usr/local/bin/nodejs
 
-ENV YARN_VERSION 0.23.2
+ENV YARN_VERSION 0.23.4
 
 RUN set -ex \
   && for key in \
diff --git a/node_boron/Dockerfile b/node_boron/Dockerfile
index 0b8a7cd..eb8aa06 100644
--- a/node_boron/Dockerfile
+++ b/node_boron/Dockerfile
@@ -21,7 +21,7 @@ RUN set -ex \
   done
 
 ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_VERSION 6.10.2
+ENV NODE_VERSION 6.10.3
 
 RUN curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-x64.tar.xz" \
   && curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/SHASUMS256.txt.asc" \
@@ -31,7 +31,7 @@ RUN curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-
   && rm "node-v$NODE_VERSION-linux-x64.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt \
   && ln -s /usr/local/bin/node /usr/local/bin/nodejs
 
-ENV YARN_VERSION 0.23.2
+ENV YARN_VERSION 0.23.4
 
 RUN set -ex \
   && for key in \
diff --git a/node_latest/Dockerfile b/node_latest/Dockerfile
index 78838cf..d626d96 100644
--- a/node_latest/Dockerfile
+++ b/node_latest/Dockerfile
@@ -21,7 +21,7 @@ RUN set -ex \
   done
 
 ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_VERSION 7.9.0
+ENV NODE_VERSION 7.10.0
 
 RUN curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-x64.tar.xz" \
   && curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/SHASUMS256.txt.asc" \
@@ -31,7 +31,7 @@ RUN curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-
   && rm "node-v$NODE_VERSION-linux-x64.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt \
   && ln -s /usr/local/bin/node /usr/local/bin/nodejs
 
-ENV YARN_VERSION 0.23.2
+ENV YARN_VERSION 0.23.4
 
 RUN set -ex \
   && for key in \
diff --git a/node_onbuild/Dockerfile b/node_onbuild/Dockerfile
index f8ededc..a21fb53 100644
--- a/node_onbuild/Dockerfile
+++ b/node_onbuild/Dockerfile
@@ -1,4 +1,4 @@
-FROM node:7.9.0
+FROM node:7.10.0
 
 RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
diff --git a/node_slim/Dockerfile b/node_slim/Dockerfile
index 330b0ef..a8ed703 100644
--- a/node_slim/Dockerfile
+++ b/node_slim/Dockerfile
@@ -21,7 +21,7 @@ RUN set -ex \
   done
 
 ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_VERSION 7.9.0
+ENV NODE_VERSION 7.10.0
 
 RUN buildDeps='xz-utils' \
     && set -x \
@@ -36,7 +36,7 @@ RUN buildDeps='xz-utils' \
     && apt-get purge -y --auto-remove $buildDeps \
     && ln -s /usr/local/bin/node /usr/local/bin/nodejs
 
-ENV YARN_VERSION 0.23.2
+ENV YARN_VERSION 0.23.4
 
 RUN set -ex \
   && for key in \
diff --git a/node_wheezy/Dockerfile b/node_wheezy/Dockerfile
index ddee61f..49ed583 100644
--- a/node_wheezy/Dockerfile
+++ b/node_wheezy/Dockerfile
@@ -21,7 +21,7 @@ RUN set -ex \
   done
 
 ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_VERSION 7.9.0
+ENV NODE_VERSION 7.10.0
 
 RUN curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-x64.tar.xz" \
   && curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/SHASUMS256.txt.asc" \
@@ -31,7 +31,7 @@ RUN curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-
   && rm "node-v$NODE_VERSION-linux-x64.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt \
   && ln -s /usr/local/bin/node /usr/local/bin/nodejs
 
-ENV YARN_VERSION 0.23.2
+ENV YARN_VERSION 0.23.4
 
 RUN set -ex \
   && for key in \
```

</details>

**Comment by yosifkit** [comment]
Build test of #2919; 18a3cb03 (`node`):

```console

$ bashbrew build node:7.10.0
Using bashbrew/cache:413552659accb6a6052055ea8cccb46011e5ffaf8030ff19da2774126b7072ff (node:7.10.0)
Tagging node:7.10.0
Tagging node:7.10
Tagging node:7
Tagging node:latest

$ test/run.sh node:7.10.0
testing node:7.10.0
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:7.10.0-alpine
Using bashbrew/cache:b1fab46b6f605d704050cd4aad84a0c5779ac9ccf0d22bcfb8081f079ea419ef (node:7.10.0-alpine)
Tagging node:7.10.0-alpine
Tagging node:7.10-alpine
Tagging node:7-alpine
Tagging node:alpine

$ test/run.sh node:7.10.0-alpine
testing node:7.10.0-alpine
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:7.10.0-onbuild
Using bashbrew/cache:c1d67e29022b977ff12d4eb1fe63574184dc15045eda034d25cc8fbf352336ef (node:7.10.0-onbuild)
Tagging node:7.10.0-onbuild
Tagging node:7.10-onbuild
Tagging node:7-onbuild
Tagging node:onbuild

$ test/run.sh node:7.10.0-onbuild
testing node:7.10.0-onbuild
	'override-cmd' [1/1]...passed


$ bashbrew build node:7.10.0-slim
Using bashbrew/cache:a421dfe6a7781cea04c0eab5db7964bb7a9a8ff6a64731b3a1a40b0640b4e225 (node:7.10.0-slim)
Tagging node:7.10.0-slim
Tagging node:7.10-slim
Tagging node:7-slim
Tagging node:slim

$ test/run.sh node:7.10.0-slim
testing node:7.10.0-slim
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:7.10.0-wheezy
Using bashbrew/cache:3e05cd0c7b7208362b0a43abe83291294c71265f05c1ae5bc8fb031238603862 (node:7.10.0-wheezy)
Tagging node:7.10.0-wheezy
Tagging node:7.10-wheezy
Tagging node:7-wheezy
Tagging node:wheezy

$ test/run.sh node:7.10.0-wheezy
testing node:7.10.0-wheezy
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:6.10.3
Using bashbrew/cache:ca413e3681202b2da70d8f443294718725c0ca5c6792f1133d4d2634ebe669ea (node:6.10.3)
Tagging node:6.10.3
Tagging node:6.10
Tagging node:6
Tagging node:boron

$ test/run.sh node:6.10.3
testing node:6.10.3
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:6.10.3-alpine
Using bashbrew/cache:188101cb213526f5e3b56ccf6b2d08f244165dbbb828181da9054a3b03535497 (node:6.10.3-alpine)
Tagging node:6.10.3-alpine
Tagging node:6.10-alpine
Tagging node:6-alpine
Tagging node:boron-alpine

$ test/run.sh node:6.10.3-alpine
testing node:6.10.3-alpine
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:6.10.3-onbuild
Using bashbrew/cache:1df8de5b043ee71e4d8e490a0f9b3530245e9715e5dc79140fa8aa8dee7b32e6 (node:6.10.3-onbuild)
Tagging node:6.10.3-onbuild
Tagging node:6.10-onbuild
Tagging node:6-onbuild
Tagging node:boron-onbuild

$ test/run.sh node:6.10.3-onbuild
testing node:6.10.3-onbuild
	'override-cmd' [1/1]...passed


$ bashbrew build node:6.10.3-slim
Using bashbrew/cache:2e825d4c051c926d40d89da73cf2bd0738167736eb8a70e15e70dc200e277439 (node:6.10.3-slim)
Tagging node:6.10.3-slim
Tagging node:6.10-slim
Tagging node:6-slim
Tagging node:boron-slim

$ test/run.sh node:6.10.3-slim
testing node:6.10.3-slim
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:6.10.3-wheezy
Using bashbrew/cache:b4d386026ee75bd64f835d4dac31a654a58b8b826dd581bb258bc3bc5996f8d9 (node:6.10.3-wheezy)
Tagging node:6.10.3-wheezy
Tagging node:6.10-wheezy
Tagging node:6-wheezy
Tagging node:boron-wheezy

$ test/run.sh node:6.10.3-wheezy
testing node:6.10.3-wheezy
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:4.8.3
Using bashbrew/cache:746de96ef9a3e4b74723dbd40a54354002b94f84d135b2809d0ca661d28886cc (node:4.8.3)
Tagging node:4.8.3
Tagging node:4.8
Tagging node:4
Tagging node:argon

$ test/run.sh node:4.8.3
testing node:4.8.3
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:4.8.3-alpine
Using bashbrew/cache:44ff871e5d39c2c84cf63b2269959fb878963588411b43d5e05f2874f53bc446 (node:4.8.3-alpine)
Tagging node:4.8.3-alpine
Tagging node:4.8-alpine
Tagging node:4-alpine
Tagging node:argon-alpine

$ test/run.sh node:4.8.3-alpine
testing node:4.8.3-alpine
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:4.8.3-onbuild
Using bashbrew/cache:2f3f5f112918aeb7c13b5a08ed4ea6da981703200b1e16794b6774747c31c373 (node:4.8.3-onbuild)
Tagging node:4.8.3-onbuild
Tagging node:4.8-onbuild
Tagging node:4-onbuild
Tagging node:argon-onbuild

$ test/run.sh node:4.8.3-onbuild
testing node:4.8.3-onbuild
	'override-cmd' [1/1]...passed


$ bashbrew build node:4.8.3-slim
Using bashbrew/cache:59dfb5ad78ae21aa073606666174979c632c8d1c8f15e6158977834ca2409d7f (node:4.8.3-slim)
Tagging node:4.8.3-slim
Tagging node:4.8-slim
Tagging node:4-slim
Tagging node:argon-slim

$ test/run.sh node:4.8.3-slim
testing node:4.8.3-slim
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:4.8.3-wheezy
Using bashbrew/cache:3f4bb629a82ff78070e63606434521019ed76ff31f80eeaeaadf12ac4b64adf7 (node:4.8.3-wheezy)
Tagging node:4.8.3-wheezy
Tagging node:4.8-wheezy
Tagging node:4-wheezy
Tagging node:argon-wheezy

$ test/run.sh node:4.8.3-wheezy
testing node:4.8.3-wheezy
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed

```

**Pull request merged**